core/src/saros/util/LineSeparatorNormalizationUtil.java

```diff
@@ -0,0 +1,78 @@
+package saros.util;
+
+import java.util.Objects;
+
+/**
+ * Utility class offering methods to normalize and denormalize text by replacing the used line
+ * separators.
+ *
+ * <p>The default line separator used for normalized content is the Unix line separator.
+ *
+ * @see #NORMALIZED_LINE_SEPARATOR
+ */
+public class LineSeparatorNormalizationUtil {
```

Comment:
how about a private constructor to avoid instances?

Reply:
:+1: Done. Will do the same for `TextPositionUtils` added in a previous PR. (As this was added in an early PR, I will probably do this in the process of merging as I can't be bothered to rebase everything AGAIN. I know that this is not the best practice, but I don't see a lot of potential to add issues with a NOP private CTOR.)